### PR TITLE
Update using-gcp.md

### PR DESCRIPTION
**Pull request by leofisG**
### Test Plan

---
This change is [Reviewable](https://reviewable.io/reviews/aptos-labs/aptos-core/1651)

### CODE REVIEW

File: developer-docs-site/docs/nodes/validator-node/using-gcp.md [hunk 1]
```diff
@@ -121,10 +121,10 @@ This will download all the Terraform dependencies for you, in the `.terraform` f
 
     ```
     aptos genesis set-validator-configuration \
-        --keys-dir ~/$WORKSPACE 
-        --local-repository-dir ~/$WORKSPACE 
-        --username <pick a username for your node> 
-        --validator-host $VALIDATOR_ADDRESS:6180 
+        --keys-dir ~/$WORKSPACE \
+        --local-repository-dir ~/$WORKSPACE \ 
+        --username <pick a username for your node> \
```
Comment (davidiw) [inline developer-docs-site/docs/nodes/validator-node/using-gcp.md:126]: nit eliminate all white space at end of lines

**Review by davidiw — COMMENTED** [review]
looks good but whitespace nit

File: developer-docs-site/docs/nodes/validator-node/using-gcp.md (hunk 1 above)
Comment (leofisG) [reply developer-docs-site/docs/nodes/validator-node/using-gcp.md]: Fixed, thanks.

**Comment by leofisG** [comment]
Also, I was having an issue following this guide when running the `aptos genesis generate-genesis`, think the `layout` file changes with more fields.

**Comment by sherry-x** [comment]
Hey @leofisG thank you for the PR! The docs for validator node installation are all updated now.

**Pull request closed without merging**